### PR TITLE
Remove `ContractAddressSalt.get_random_value()` usages

### LINKED ISSUE #585: Replace `ContractAddressSalt.get_random_value()` with our implementation
### Feature Request

Consider the security of the new approach

### PR DESCRIPTION
**Pull request by war-in**
Closes #585 

## Introduced changes

Use `get_random_salt()` from utils/crypto/facade.py instead of `ContractAddressSalt.get_random_value()`

##

- [ ] This PR contains breaking changes

### CODE REVIEW

File: starknet_py/utils/crypto/facade.py [hunk 1]
```diff
@@ -46,3 +48,7 @@ def pedersen_hash(left: int, right: int) -> int:
     if use_cpp_variant():
         return cpp_hash(left, right)
     return default_hash(left, right)
+
+
+def get_random_salt() -> int:
+    return random.Random().randrange(0, FIELD_PRIME)
```
Comment (drknzz) [inline starknet_py/utils/crypto/facade.py:54]: Shouldn't FIELD_PRIME also be included?

Comment (drknzz) [reply starknet_py/utils/crypto/facade.py:54]: I've checked, it should not.

**Review by drknzz — APPROVED** [review]
👍

File: starknet_py/utils/crypto/facade.py (hunk 1 above)
Comment (Solpatium) [inline starknet_py/utils/crypto/facade.py:54]: This doesn't have anything with crypto, it doesn't use entropy based random values now. I think we can move it somewhere else.

**Review by Solpatium — CHANGES_REQUESTED** [review]

**Review by cptartur — COMMENTED** [review]
Agreed with @Solpatium comment

File: starknet_py/utils/crypto/facade.py (hunk 1 above)
Comment (war-in) [reply starknet_py/utils/crypto/facade.py]: Does everyone agree that `get_random_salt` should be moved to the `udc_deployer/deployer.py` file? I'm quite unsure, but this is the only place where we use it (outside the tests)

**Review by cptartur — APPROVED** [review]

File: starknet_py/net/udc_deployer/deployer.py [hunk 2]
```diff
@@ -123,3 +125,7 @@ def _compute_address(
             salt=salt,
             deployer_address=deployer_address,
         )
+
+
+def get_random_salt() -> int:
```
[the remark this answers is not in the document]
Comment (war-in) [reply starknet_py/net/udc_deployer/deployer.py]: And it is :hmmm: 
image

**Review by Solpatium — APPROVED** [review]

**Pull request merged**